dev-ml/parsexp/metadata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="project">
+    <email>ml@gentoo.org</email>
```

Comment:
Do you have their consent that they are willing to maintain this? Looking at the activity on bug / PR assigned to them...

You need to maintain it by yourself, along with the proxy-maint project unless you have indeed received some acknowledgment from the ml project.